app/src/main/java/com/babylon/wallet/android/presentation/transaction/TransactionReviewViewModel.kt

```diff
@@ -494,7 +480,7 @@ sealed interface PreviewType {
                 Contribution, Redemption
             }
 
-            val poolsInvolved: Set<com.babylon.wallet.android.domain.model.resources.Pool>
+            val poolsInvolved: Set<rdx.works.core.domain.resources.Pool>
```

Comment:
not import?

Reply:
It clashes with the name of the Pool PreviewType